Formula/bgfx.rb

```diff
@@ -0,0 +1,74 @@
+class Bgfx < Formula
+  desc "Cross-platform, graphics API agnostic, rendering library"
+  homepage "https://github.com/bkaradzic/bgfx"
+  url "https://github.com/bkaradzic/bgfx/archive/403b69db7876bc7269f546273bba57c40d2b137e.tar.gz"
+  version "1.115.7816"
+  sha256 "f6808e6011e93470d1c60be6d371e905e5c703067dc0ee21b7c8943c60f8e042"
+  license "BSD-2-Clause"
+  head "https://github.com/bkaradzic/bgfx.git", branch: "master"
+
+  depends_on "cmake" => :build
+
+  on_linux do
+    depends_on "mesa"
+  end
+
```

Comment:
```suggestion

  fails_with gcc: "5"
```
This might help with the Linux error.